bootstrap/events/consumer/streams.go

```diff
@@ -44,34 +44,34 @@ func (es *eventHandler) Handle(ctx context.Context, event events.Event) error {
 
 	switch msg["operation"] {
 	case thingRemove:
```

Comment:
The value of thingRemove is `thing.remove` , this should be changed to `client.remove`